code/script.js

```diff
@@ -0,0 +1,77 @@
+const USER = "waliem"
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`
+const USER_INFO_URL = `https://api.github.com/users/${USER}`
+
+const projectsContainer = document.getElementById("projects")
+const userContainer = document.getElementById("user-info")
+
+// function for getting my user info
+const getUserInfo = () => {
+  fetch(USER_INFO_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      userContainer.innerHTML += `
+      <img class="profile-img" src="https://avatars.githubusercontent.com/u/84201089?v=4" alt="User profile picture">
+      <span id="profile-name"> <a class="usr-name" href="https://github.com/waliem"><h2>${data.login}</h2></a> <h3>Front end developer student</h3><h3>Based in Gothenburg</h3></span>`
```

Comment:
You could also fetch this info (the ones you have on your GitHub) through the API - ex. data.html_url for the profile link.